src/components/covidInformation/index.tsx

```diff
@@ -16,18 +16,37 @@ const CovidInformation = ({
   group?: string | undefined;
 }): React.ReactElement => {
   return (
-    <>
-      <p>Data that got fetched: </p>
+    <div className="mb-12">
+      <p className="mb-6 text-sm font-normal text-textColor">
+        Data that got fetched
+      </p>
       <ul>
-        {group ? <li> Workplace: {group} </li> : null}
-        <li> Type of certificate: {certificaat} </li>
-        <li>
-          Date: {date} {endDate ? `- ${endDate}` : null}
+        {group ? (
+          <li className="text-md text-navyBlue">
+            <span className="mr-2 text-textColor"> Workplace: </span> {group}
+          </li>
```

Comment:
Misschien een componentje van maken met altijd die styling. Als je dan de styling wilt aanpassen moet je dat maar 1x doen ipv 4x in dit geval.